esp-hal-common/rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
```

Comment:
we generally don't check-in this file for esp-hal-common - probably it should be added to `.gitignore`

Reply:
Ah yes I forgot to remove it, I'll also add it to gitignore